### PR TITLE
appwrite cli login issue - Update client.js.twig

### PR DESCRIPTION
**Pull request by StarZeus**
This change enables appwrite-cli to capture all cookies in 'set-cookie' response headers.

Fixes https://github.com/appwrite/sdk-for-cli/issues/110

### CODE REVIEW

File: templates/cli/lib/client.js.twig [hunk 1]
```diff
@@ -154,7 +154,11 @@ class Client {
 
     let cookies = response.headers.getSetCookie();
     if (cookies && cookies.length > 0) {
-      globalConfig.setCookie(cookies[0]);
+      let cookieString = "";
+      cookies.forEach((cookie) => {
+          cookieString += cookie.split(";")[0] + ";";
```
Comment (stnguyen90) [inline templates/cli/lib/client.js.twig:159]: Why do we need to do this split?

Comment (lohanidamodar) [reply templates/cli/lib/client.js.twig:159]: I think this makes sense, if first cookie is set to something else, doing this will save all the kookies.

Comment (Meldiron) [reply templates/cli/lib/client.js.twig:159]:
```suggestion
      globalConfig.setCookie(cookies.join(";"));
```
Would this do the job? 🤔

Comment (StarZeus) [reply templates/cli/lib/client.js.twig:159]: That sounds simpler :). Thanks @Meldiron. I'll update the PR.

**Comment by StarZeus** [comment]
@stnguyen90 - Updated the PR. Please let me know if you have any questions.

**Review by stnguyen90 — APPROVED** [review]

**Comment by stnguyen90** [comment]
LGTM!

image

**Pull request merged**